docs/category-overview-pages/accessing-netdata-dashboards.md

```diff
@@ -23,5 +23,17 @@ The Netdata dashboard consists of the following main sections:
 
 > ⚠️ Some sections of the dashboard, when accessed through the agent,  may require the user to be signed in to Netdata Cloud or having the Agent claimed to Netdata Cloud for their full functionality. Examples include saving visualization settings on charts or custom dashboards, claiming the node to Netdata Cloud, or executing functions on an Agent.
 
+## How to access the dashboards?
+
+### Netdata Cloud
+
+You can access the dashboard at https://app.netdata.cloud/ and [sign-in](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/sign-in.md) with an account or [sign-up](https://github.com/netdata/netdata/blob/master/docs/cloud/manage/sign-in.md#dont-have-a-netdata-cloud-account-yet) if you don't have an account yet.
+
+### Netdata Agent
+
+Netdata starts a web server for its dashboard at port `19999`. Open up your web browser of choice and
+navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If you're unsure, try
+`http://localhost:19999` first.
```

Comment:
```suggestion
navigate to `http://NODE:19999`, replacing `NODE` with the IP address or hostname of your Agent. If installed on localhost, you can access it through `http://localhost:19999`.
```

Reply:
this is already on this commit. right?

https://github.com/netdata/netdata/pull/15925/commits/4097f8685482198b319d104e3afe3c3440940157#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R169

if you are ok lets resolve the conversation

Reply:
It is not, you haven't changed `docs/category-overview-pages/accessing-netdata-dashboards.md`.